pkg/connector/app.go

```diff
@@ -176,14 +180,15 @@ func (o *appResourceType) listAppGroupGrants(
 	}
 
 	for _, applicationGroupAssignment := range applicationGroupAssignments {
-		roles, _, err := o.client.Group.ListGroupAssignedRoles(ctx, applicationGroupAssignment.Id, nil)
-		if err != nil {
-			return nil, annos, bag, err
-		}
-
-		for _, role := range roles {
-			rv = append(rv, appGroupGrant(resource, applicationGroupAssignment, role.Type))
-		}
+		groupID := applicationGroupAssignment.Id
+		principalID := &v2.ResourceId{ResourceType: resourceTypeGroup.Id, Resource: groupID}
+		rv = append(rv, sdkGrant.NewGrant(resource, "access", principalID,
+			sdkGrant.WithAnnotation(
+				&v2.V1Identifier{
+					Id: fmtGrantIdV1(V1MembershipEntitlementID(resource.Id.Resource), groupID),
+				},
+			),
+		))
```

Comment:
Doesn't this change return way fewer grants? It's probably why CI is failing, because we're granting/revoking a user's super admin on an app.